src/app/static/src/app/components/baseline/Baseline.vue

```diff
@@ -3,34 +3,34 @@
         <div class="row">
             <div class="col-sm-6 col-md-8">
                 <form>
-                    <file-upload label="PJNZ"
-                                 :valid="pjnz.valid"
-                                 :error="pjnz.error || plottingMetadataError"
-                                 :upload="uploadPJNZ"
-                                 :delete-file="deletePJNZ"
-                                 :existingFileName="pjnz.existingFileName"
-                                 accept="PJNZ,pjnz,.pjnz,.PJNZ,.zip,zip,ZIP,.ZIP"
-                                 name="pjnz">
+                    <manage-file label="PJNZ"
+                                     :valid="pjnz.valid"
+                                     :error="pjnz.error || plottingMetadataError"
+                                     :upload="uploadPJNZ"
+                                     :delete-file="deletePJNZ"
+                                     :existingFileName="pjnz.existingFileName"
+                                     accept="PJNZ,pjnz,.pjnz,.PJNZ,.zip,zip,ZIP,.ZIP"
+                                     name="pjnz">
                         <label v-if="country"><strong v-translate="'country'"></strong>: {{country}}</label>
-                    </file-upload>
-                    <file-upload label="shape"
+                    </manage-file>
+                    <manage-file label="shape"
```

Comment:
Are these file types called shape by convention? Would it perhaps be less confusing for the user to call it  a geojson file to make the file that needs to be uploaded more explicit? I realise the change would need to be made in the translation file, not here though.

Reply:
Terminology is guided by Jeff/UNAIDs, so far this seems to have been the convention!